app/scripts/modules/amazon/loadBalancer/configure/createLoadBalancer.controller.js

```diff
@@ -235,6 +235,24 @@ module.exports = angular.module('spinnaker.loadBalancer.aws.create.controller',
       $scope.existingSecurityGroupNames = [];
     }
 
+    function certificateIdAsARN(accountId, certificateId) {
+      if (certificateId) {
+        // If they really want to enter the ARN...
+        if (!certificateId.indexOf('arn') > -1) {
```

Comment:
Can the `certificateId` contain `arn` without starting with `arn`? I honestly don't know much about what the IDs look like, just wondering if this should be `=== 0` instead of `> -1`.